osu.Game/Screens/Play/GameplayState.cs

```diff
@@ -40,6 +41,11 @@ public class GameplayState
 
         public readonly ScoreProcessor ScoreProcessor;
 
+        /// <summary>
+        /// The storyboard associated with the beatmap.
+        /// </summary>
+        public readonly Storyboard Storyboard;
```

Comment:
not 100% on a storyboard existing here, but it's not causing any short-term harm.